raiden/console.py

```diff
@@ -228,8 +227,8 @@ def ping(self, peer):
             print("Error: peer {} not found in discovery".format(peer))
             return
 
-        nonce = self._ping_nonces[peer]
-        self._ping_nonces[peer] += 1
+        nonce = self._raiden.protocol.ping_nonces[peer]
```

Comment:
I would prefer keeping PRs as small as possible and to the point and definitely will not combine API changes here. But yes I see your 'otherwise' suggestion as something that should be done in this PR.